.env.local.sample

```diff
@@ -35,3 +35,5 @@ MAIL_PASSWORD=
 MAIL_FROM_ADDRESS=staging@terramatch.org
 
 TRANSIFEX_TOKEN=
+
+APP_BACKEND_URL=https://new-wri-prod.wri-restoration-marketplace-api.com
```

Comment:
This file should have a value that makes sense for local development.

Reply:
Also, I just found where this is being used. We should go ahead and make this use the API gateway. A change was made to the CDK stack for the gateway recently to send requests for `images/v2` to the PHP BE (https://github.com/wri/terramatch-microservices/blob/staging/cdk/api-gateway/lib/api-gateway-stack.ts#L95C1-L95C87). I think if that is simply changed to `images` (dropping the V2), then this can be `api.terramatch.org` for prod, `api-staging.terramatch.org` for staging, etc. That will probably help keep our emails from getting flagged as spam too, since then the image domain will match the from domain on the email.

Reply:
Please go ahead and make this change in this PR.